2020-12-28 20:29:03,343 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 20:54:20,007 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 20:57:18,277 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:02:50,722 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:05:59,488 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:12:56,428 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:16:36,825 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:40:53,149 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:44:18,068 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:46:09,504 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:47:04,556 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:47:50,673 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:48:44,788 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 21:51:06,172 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2020-12-28 22:04:34,166 com_data.py[line:35] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
